flows/actions/send_msg.go

```diff
@@ -47,7 +47,7 @@ type SendMsgAction struct {
 	createMsgAction
 
 	AllURNs           bool                      `json:"all_urns,omitempty"`
-	Template          *assets.TemplateReference `json:"template,omitempty"`
+	Template          *assets.TemplateReference `json:"template,omitempty" validate:"omitempty"`
```

Comment:
if this was necessary... every single `send_msg` action without a template would be failing .. no?

Reply:
Yes, I think those that are failing are those with a previous version set before they migrate to the latest version

Reply:
And why are we only seeing this problem for simulation?

Reply:
mailroom will use the flow definition passed from the simulation request and that is the definition before the migrations I think